custom.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+
+var character;
+var currentCharacter = [];
+var myCharacters = [];
+// var slugbyte = new Character('slugbyte', 'human', 'male', 'wizard', 'chaoticGood');
+// currentCharacter.push(slugbyte);
+// localStorage.setItem('myCharacters', JSON.stringify(myCharacters));
+// localStorage.setItem('currentCharacter', JSON.stringify(currentCharacter));
+myCharacters = JSON.parse(localStorage.getItem('myCharacters'));
+currentCharacter = JSON.parse(localStorage.getItem('currentCharacter'));
+
+if (currentCharacter < 1){
+  currentCharacter = myCharacters[0];
+}
+else if (currentCharacter === null){
+  currentCharacter = [];
+}
+
+
+function Character(name, race, gender, charClass, align) {
```

Comment:
Yall should not have two constructors that model the same exact thing
this goes agains Dont Repeat Yourself (DRY) coding